OpenOversight/app/templates/add_edit_department.html

```diff
@@ -15,10 +16,49 @@ <h1>{% if update %} Update {% else %} Add {% endif %} Department</h1>
 
         {{ wtf.form_field(form.name, autofocus="autofocus") }}
         {{ wtf.form_field(form.short_name) }}
+        <div>
+          <legend>{{ form.ranks.label }}</legend>
+          <div>
+            <h5>Enter ranks in hierarchical order:</h5>
+          </div>
+          <div class="text-danger">
+            {{ wtf.form_errors(form.ranks, hiddens="only") }}
+          </div>
+          {% if form.ranks|length > 1 %}
+            {% for subfield in (form.ranks|sort(attribute='data.order'))[1:] %}
+            <fieldset>
+              <div class="input-group {% if subfield.errors %} has-error{% endif -%}
+                         {%- if subfield.flags.required %} required{% endif -%}">
+                {{ subfield(class="form-control")|safe}}
+                <span class="input-group-btn">
+                  <button class="btn btn-danger js-remove-button" style="margin-bottom: 2rem;" disabled>Remove</button>
+                </span>
+                {%- if subfield.errors %}
+                  {%- for error in subfield.errors %}
+                    <p class="help-block">{{error}}</p>
+                  {%- endfor %}
+                {%- endif %}
+              </div>
+            </fieldset>
+            {% endfor %}
+          {% else %}
+            <fieldset>
+              <div class="form-group ">
+                <input class="form-control" custom_name="ranks-1" id="ranks-1" name="ranks-1" type="text" value="">
+              </div>
+              <button class="btn btn-danger js-remove-button" style="margin-bottom: 2rem;">Remove</button>
```

Comment:
I think there is a bug here, check out the following STR:

0. Go to add a "New Department"
1. Enter first rank name.
2. Click add a new rank and enter a second rank name.
3. Click remove on the new rank.

I would expect that just the most recently added rank would disappear, but the entire rank section disappears

Reply:
(I think this is not yet resolved, lmk if I'm wrong or you were unable to repro)